exercises/practice/hello-world/.meta/config.json

```diff
@@ -1,6 +1,53 @@
 {
   "blurb": "The classical introductory exercise. Just say \"Hello, World!\"",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/0b77b05fd7a97b7819f29c0d32e436e600092607#diff-455eb2dc457644ff50c312dd8d72cb57e92595d26f13f5c9a94bfd9e78b4a36c by Rob Phoenix (other person then the other empty ones, but also without a github account)